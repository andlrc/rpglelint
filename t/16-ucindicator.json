[
   {
      "column":8,
      "file":"t/16-ucindicator.rpgle",
      "line":2,
      "message":"indicator '*oN' needs to be all uppercase",
      "severity":"warning"
   },
   {
      "column":7,
      "file":"t/16-ucindicator.rpgle",
      "line":3,
      "message":"indicator '*OfF' needs to be all uppercase",
      "severity":"warning"
   },
   {
      "column":7,
      "file":"t/16-ucindicator.rpgle",
      "line":5,
      "message":"indicator '*NUll' needs to be all uppercase",
      "severity":"warning"
   },
   {
      "column":8,
      "file":"t/16-ucindicator.rpgle",
      "line":7,
      "message":"indicator '*blank' needs to be all uppercase",
      "severity":"warning"
   },
   {
      "column":23,
      "file":"t/16-ucindicator.rpgle",
      "line":7,
      "message":"indicator '*blank' needs to be all uppercase",
      "severity":"warning"
   }
]
