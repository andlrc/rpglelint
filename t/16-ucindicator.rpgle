// rpglelint: -Wucindicator
if a = *oN;
  b = *OfF;
else;
  c = *NUll;
endif;
if d = *blank and e = *blanks;
  *INLR = *ON;
endif;

if *In10;
  doIt();
endif;
