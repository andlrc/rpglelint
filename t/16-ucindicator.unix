t/16-ucindicator.rpgle:2:8: warning: indicator '*oN' needs to be all uppercase [-Wuppercase-indicator]
 if a = *oN;
        ^
t/16-ucindicator.rpgle:3:7: warning: indicator '*OfF' needs to be all uppercase [-Wuppercase-indicator]
   b = *OfF;
       ^
t/16-ucindicator.rpgle:5:7: warning: indicator '*NUll' needs to be all uppercase [-Wuppercase-indicator]
   c = *NUll;
       ^
t/16-ucindicator.rpgle:7:8: warning: indicator '*blank' needs to be all uppercase [-Wuppercase-indicator]
 if d = *blank and e = *blanks;
        ^
t/16-ucindicator.rpgle:7:23: warning: indicator '*blank' needs to be all uppercase [-Wuppercase-indicator]
 if d = *blank and e = *blanks;
                       ^
