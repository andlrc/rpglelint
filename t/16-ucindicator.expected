t/16-ucindicator.rpgle:2:8: warning: indicator '*oN' needs to be all uppercase [-Wucindicator]
 if a = *oN;
        ^
t/16-ucindicator.rpgle:3:7: warning: indicator '*OfF' needs to be all uppercase [-Wucindicator]
   b = *OfF;
       ^
t/16-ucindicator.rpgle:5:7: warning: indicator '*NUll' needs to be all uppercase [-Wucindicator]
   c = *NUll;
       ^
t/16-ucindicator.rpgle:7:8: warning: indicator '*blank' needs to be all uppercase [-Wucindicator]
 if d = *blank and e = *blanks;
        ^
t/16-ucindicator.rpgle:7:23: warning: indicator '*blank' needs to be all uppercase [-Wucindicator]
 if d = *blank and e = *blanks;
                       ^
